cassovary-core/src/main/scala/com/twitter/cassovary/util/collections/FastQueue.scala

```diff
@@ -0,0 +1,119 @@
+package com.twitter.cassovary.util.collections
+
+import it.unimi.dsi.fastutil.PriorityQueue
+import it.unimi.dsi.fastutil.ints.{IntComparator, IntHeapPriorityQueue, IntArrayFIFOQueue, IntPriorityQueue}
+import it.unimi.dsi.fastutil.objects.{ObjectArrayFIFOQueue, ObjectArrayPriorityQueue}
+
+abstract class FastQueue[@specialized(Int, Long) T] {
```

Comment:
I wanted to avoid fastutils in all traversers code. For this we needed Queue too. Maybe we can leave Queue here and put rest in separate commits?